### DISCUSSION TITLE
Separate Plugins config to own plugins.xml

### DISCUSSION THREAD

**Discussion by timur-g** [doublecmd/doublecmd#1107] [Suggestions]
### Description

DC configuration is mostly in doublecmd.xml, with some additional xml files.
For us switching to another system, and also Win-Lin, just taking doublecmd.xml is great, except for pugins.
Now plugins configuration is within doublecmd.xml and it makes a problem if using the same doublecmd.xml for Win-Lin.

### Proposed technical implementation details (optional)

IMO plugins conf should have it's own plugins.xml, like extassoc.xml, so it would be different for Win and Lin (\ and /).
Moreover, I'd suggest having separate plugins-L.xml and plugins-W.xml, so that we sharing DC configuration can keep both in the same directory.
That could be done also for other confs depending on OS: extassoc.xml.

**Comment by ags1234** [comment]
Free Commander is nicely organised in regards to the config files.
Everything, every config file, is in a directory called Settings. When a new version is released, just paste this directory and you feel right at home.

It would be great if DC would have all it's personalization files in a separate directory.
I know it is somehow  traditional to have the config file next to the exe, but when you have more files with different configs, they should be grouped together.
